### PR TITLE
high resolution vtm logo

### PR DESCRIPTION
**Pull request by Gustl22**
add vtm logo as vector drawable and update `ic_launcher` icon in android samples.

### CODE REVIEW

**Comment by devemux86** [comment]
Probably a blue with more contrast with gray background could be used?

**Comment by Gustl22** [comment]
Yes that was my intention, too :)

**Comment by Gustl22** [comment]
* brighter blue for vtm logo
* improve OSciMap logo
* add logo to desktop gdx app
* add logo to web app (favicon in tab)

Feel free to rename resources or packages/directories or move them in the right package.

File: vtm-desktop/src/org/oscim/gdx/GdxMapApp.java [hunk 1]
```diff
@@ -62,6 +63,14 @@ protected static LwjglApplicationConfiguration getConfig(String title) {
         LwjglApplicationConfiguration.disableAudio = true;
         LwjglApplicationConfiguration cfg = new LwjglApplicationConfiguration();
         cfg.title = title != null ? title : "vtm-gdx";
+
+        String[] imgDimensions = new String[]{"16", "32", "64", "128"};
+        for (String imgDimension : imgDimensions) {
+            String path = "assets/launcher/ic_vtm_" + imgDimension + ".png";
+            //if (Gdx.files.internal(path).exists()) // Check if file exist!
```
Comment (Gustl22) [inline vtm-desktop/src/org/oscim/gdx/GdxMapApp.java]: I'm not sure if icons are detected in any system/environment. So I wanted to check if file exists, but this isn't possible with gdx tools as they aren't initialized at this moment.

File: vtm-web-app/war/index.html [hunk 2]
```diff
@@ -2,6 +2,7 @@
 <html>
 <head>
     <title>vtm-gdx</title>
+    <link rel="icon" type="image/svg+xml" href="assets/launcher/ic_vtm.svg" sizes="any">
```
Comment (devemux86) [inline vtm-web-app/war/index.html]: That svg is missing?

Comment (Gustl22) [reply vtm-web-app/war/index.html]: Yes, `/vtm-web-app/assets/launcher` was in .gitignore. I added it now to the vtm assets, cause I think it is logo for all modules (?). `vtm-themes/resources/assets` or `/vtm-web-app/war` maybe possible, too...

Comment (devemux86) [reply vtm-web-app/war/index.html]: `vtm-web-app/assets` is constructed from all modules during web app build.
I moved the svg in `vtm-web-app/resources/assets/logo/` where web app assets exist.

**Comment by Gustl22** [comment]
@Longri if you want, can add icons to iOS after merge? Can say me the e needed sizes if they doesn't exist already in `vtm-android-example` project.

**Comment by devemux86** [comment]
Thanks, nice work!
Squashed and merged via 77d0c36539c9b8f3a1dcc37ce0877afffe0af900.

Included only the svg in docs, as much larger eps could not be needed?

**Pull request closed without merging**